activestorage/app/models/active_storage/blob.rb

```diff
@@ -350,7 +350,8 @@ def compute_checksum_in_chunks(io)
       raise ArgumentError, "io must be rewindable" unless io.respond_to?(:rewind)
 
       OpenSSL::Digest::MD5.new.tap do |checksum|
-        while chunk = io.read(5.megabytes)
+        buffer = +""
```

Comment:
Although it may not matter to `OpenSSL::Digest::MD5#<<`, I think it would be safer to ensure a binary (`ASCII_8BIT`) encoding here:

```suggestion
        buffer = "".b
```

Reply:
@jonathanhefner Applied the changes.